SU-HSDO/suhumsci
# STN-556 : Rollback Carousel and Hero Refinements

# READY FOR REVIEW

## Summary
Rollback carousel and hero refinements from sprint 24.
Reincorporate alignment fix of carousel controls in the Colorful themed carousel.

## Need Review By (Date)
ASAP

## Urgency
high 🔥

## Steps to Test
1. In you CLI, run `npm run test` and confirm our tests pass.
2. Review rollback of Colorful and Traditional carousels: http://sparkbox-sandbox.suhumsci.loc/qa/carousel-hero-image
3. Review rollback of Colorful and Traditional heros: http://sparkbox-sandbox.suhumsci.loc/qa/hero-height-refinements. Note that the heros have been rolled back to exactly what is on production. Which means that a large amount of content can/will overflow the hero area. On the Traditional theme, the content block also does not align consistently. This is especially noticeable on heros within the main content area of the page.
